benchmarks/test/test_discovery.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""@package integration_test.py
+
+Just a demo for now. Show how to tap into Nighthawk's
+integration test framework to run benchmark executions.
+"""
+
+import logging
+import json
+import pytest
+import os
+from test.integration.integration_test_fixtures import (http_test_server_fixture,
+                                                        https_test_server_fixture)
+from test.integration.utility import *
+from envoy_proxy import (inject_envoy_http_proxy_fixture, proxy_config)
+
+
+def run_with_cpu_profiler(fixture,
```

Comment:
Why is this in discovery? Isn't this just general execution ochestration?

Reply:
I may want to pull this out of this PR and host it as an example on a private repo or some such. WDYT?

Reply:
Refactoring makes sense, or are you saying the whole file is moved somewhere?

Reply:
I was suggesting the possibility of moving it out of here and hosting it somewhere else. But after thinking about it: let's keep it in here, I'll see if I can move it into a more sensible place then. We will eventually need a location that hosts "foreign" tests for testing the full docker flow though.